### PR TITLE
Simple feed

### LINKED ISSUE #3: Research and Integrate RSS api
5 hour

simple requests for api calls

### LINKED ISSUE #4: Get Simple feed from a news host
5 hour

Get RSS feed from 1 news host and parse it to be sent in json format

### PR DESCRIPTION
**Pull request by akwong189**
Working simple feed wrapper which closes #3 and closes #4

### CODE REVIEW

**Comment by Kyle-McRae** [comment]
Very clean and readable!

**Comment by sfparrett** [comment]
Looks very good! The run.sh script is definitely the way to go!

File: sample_backend.py [hunk 1]
```diff
@@ -28,3 +28,31 @@ def get_categoy_bubble_data():
 
 # @app.route('/users/<id>/<job>',methods = ['GET'])
 # def get_users_with_job(id,job):
+@app.route('/feed', methods=['GET'])
+def get_app_feed():
+   """ Get custom feed from NewsAPI and return it """
+   if request.method == 'GET':
+      q = request.args.get("q")
+      q_in_title = request.args.get("qInTitle")
```
Comment (sfparrett) [inline sample_backend.py:36]: A lot of repeated code here maybe this can be simplified a bit?

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```
Comment (sfparrett) [inline run.sh:2]: This file is a great idea, a way better way to run our backend code!

File: feed.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+class NewsAPICalls:
+    """ Wrapper class for NewsAPI calls """
+
+    def __init__(self):
+        config = configparser.ConfigParser()
+        config.read('.config/api.conf')
+        self.api_key = config['NewsAPI']['key'].strip("\'")
+
+    def get_requests(self, url: str, data=None):
+        """ helper function, send a get request to a specified url and any parameters """
```
Comment (sfparrett) [inline feed.py:23]: We should probably add some error handling to these feed functions (Some try/except logic + error logging)

**Review by sfparrett — COMMENTED** [review]
Overall looks good! Nice job refactoring.

**Pull request merged**